### PR TITLE
Create items of home screen

### PR DESCRIPTION
**Pull request by tinvongoc54**
Create items of home screen

- https://edu-redmine.framgia.vn/issues/7417

### CODE REVIEW

**Comment by tinvongoc54** [comment]
ready

**Comment by tuanlvt-1117** [comment]
fix conflict

**Comment by tuanlvt-1117** [comment]
commented

**Comment by tinvongoc54** [comment]
ready

File: app/src/main/res/drawable/custom_main_background.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:angle="90"
+        android:endColor="@color/colorEndMainBackground"
+        android:startColor="@color/colorStartMainBackground"/>
+</shape>
```
Comment (tuanlvt-1117) [inline app/src/main/res/drawable/custom_main_background.xml]: enter end line

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,8 +1,98 @@
...
+    <TextView
+        android:id="@+id/text_city_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="10dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="10dp"
+        android:gravity="center"
+        android:text="@string/city_name"
+        android:textSize="40sp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: hard code.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,8 +1,98 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+    <TextView
+        android:id="@+id/text_city_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="10dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="10dp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: hard code.

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,8 +1,98 @@
...
+        android:layout_height="wrap_content"
+        android:background="?attr/colorPrimary"
+        android:minHeight="?attr/actionBarSize"
+        android:theme="?attr/actionBarTheme"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+    <TextView
+        android:id="@+id/text_city_name"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
        android:id="@+id/textCityName"
```

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -1,8 +1,98 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/toolbar"
+        />
+    <TextView
+        android:id="@+id/text_current_date"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="10dp"
+        android:layout_marginStart="10dp"
+        android:text="@string/current_date"
+        android:textSize="20sp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml]: hard code

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -1,8 +1,98 @@
...
+        />
+    <TextView
+        android:id="@+id/text_hourly"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:layout_marginEnd="10dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="10dp"
+        android:text="@string/hourly"
+        android:textSize="20sp"
```
Comment (phuongohn-1626) [inline app/src/main/res/layout/activity_main.xml]: hard code

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -1,8 +1,98 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/recyclerViewDaily"
+        />
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/recyclerViewHourly"
+        android:layout_width="0dp"
+        android:layout_height="120dp"
+        android:layout_marginEnd="10dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="10dp"
```
Comment (phuongohn-1626) [inline app/src/main/res/layout/activity_main.xml]: hard code

File: app/src/main/res/menu/menu_setting.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:id="@+id/action_save_location"
+        android:title="@string/save_location"/>
+    <item
+        android:id="@+id/action_setting"
+        android:title="@string/setting"/>
+</menu>
```
Comment (phuongohn-1626) [inline app/src/main/res/menu/menu_setting.xml]: enter end line

File: app/src/main/res/layout/activity_main.xml [hunk 9]
```diff
@@ -1,8 +1,98 @@
...
+        />
+    <TextView
+        android:id="@+id/text_daily"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/daily"
+        android:textSize="20sp"
```
Comment (phuongohn-1626) [inline app/src/main/res/layout/activity_main.xml]: hard code

**Comment by tinvongoc54** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 10]
```diff
@@ -1,8 +1,98 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/custom_main_background"
+    tools:context=".screen.main.MainActivity"
     >
 
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="@dimen/dp_0"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/activity_main.xml:14]: 0dp không cần đưa vào dimen đâu e

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**